config/features.yml

```diff
@@ -330,10 +330,6 @@ features:
     actor_type: user
     description: Manage dependent removal from view dependent page
     enable_in_development: true
-  dependents_encrypt_jobs:
-    actor_type: user
-    description: Manage whether Sidekiq Jobs used for Dependents Claims will encrypt arguments considered potential PII.
-    enable_in_development: true
```

Comment:
Once this is deployed, you'll need to follow step 3 of [this section of this guide](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#Featuretogglesguide-Removingold/unusedfeaturetoggles) to completely remove this toggle. You'll need to do it in the terminal of each ArgoCD environment. If you or no one on your team has access to those terminals, let me know and I can do it for you. Remember, don't delete them in Argo until the code has been deployed.